packages/expo-clipboard/src/web/WebExpoClipboard.ts

```diff
@@ -0,0 +1,136 @@
+import {
+  ClipboardImage,
+  GetImageOptions,
+  GetStringOptions,
+  SetStringOptions,
+} from '../Clipboard.types';
+import {
+  ClipboardUnavailableException,
+  CopyFailureException,
+  NoPermissionException,
+  PasteFailureException,
+} from './WebExceptions';
+import {
+  base64toBlob,
+  blobToBase64Async,
+  findImageInClipboardAsync,
+  getImageSizeFromBlobAsync,
+  isClipboardPermissionDeniedAsync,
+} from './WebUtils';
+
+export default {
+  get name(): string {
+    return 'ExpoClipboard';
+  },
+  async getStringAsync(_options: GetStringOptions): Promise<string> {
+    let text = '';
+    try {
+      text = await navigator.clipboard.readText();
+    } catch (e) {
+      // it might fail, because user denied permission
+      if (e.name === 'NotAllowedError' || (await isClipboardPermissionDeniedAsync())) {
+        throw new NoPermissionException();
+      }
+
+      try {
+        // Internet Explorer
+        // @ts-ignore
+        text = window.clipboardData.getData('Text');
+      } catch (e) {
+        return Promise.reject(new Error('Unable to retrieve item from clipboard.'));
+      }
+    }
+    return text;
+  },
+  setString(text: string): boolean {
+    let success = false;
+    const textField = document.createElement('textarea');
+    textField.textContent = text;
+    document.body.appendChild(textField);
+    textField.select();
+    try {
+      document.execCommand('copy');
+      success = true;
+    } catch (e) {}
+    document.body.removeChild(textField);
+    return success;
```

Comment:
👀 This way is not a good option? 🤔 
It's just a suggestion, but I wonder whether we should we more strict about not using such variables `let success = false` and modyfying them later.
```suggestion
    try {
      document.execCommand('copy');
      return true
    } catch (e) {
      return false
    } finally {
      document.body.removeChild(textField);
    }
```

Reply:
Old code (not mine 😅 ), but a good catch 👍